src/common/src/config.rs

```diff
@@ -595,6 +595,18 @@ pub struct CacheRefillConfig {
     #[serde(default = "default::cache_refill::concurrency")]
     pub concurrency: usize,
 
+    #[serde(default = "default::cache_refill::unit")]
+    pub unit: usize,
```

Comment:
It is better to add some comments about this config since its name is not very instructive like other configs.